test/README.md

```diff
@@ -108,17 +108,23 @@ tests will fail. If this happens, remove the cache directory (and make
 sure bitcoind processes are stopped as above):
 
 ```bash
-rm -rf cache
+rm -rf test/cache
 killall bitcoind
 ```
 
 ##### Test logging
 
+The test framework logs to two files: 
+ * `<test data directory>/test_framework.log` 
+ * `<test data directory>/<node>/regtest/debug.log`. 
+ 
+The `<test data directory>` path is by default set to the temporary directory of your OS, and its full path is written as the very first line of the test output. The `<node>` part identifies the relevant test node (from `node1` to `node4`).  
```

Comment:
Would be nice to stay within the max line length seen in the file.

Reply:
Agreed! Fixed.